actors/miner/src/lib.rs

```diff
@@ -1432,10 +1409,7 @@ impl Actor {
 
             // Overwrite sector infos.
             sectors.store(new_sectors).map_err(|e| {
-                e.downcast_default(
-                    ExitCode::USR_ILLEGAL_STATE,
-                    "failed to update sector infos",
-                )
+                e.downcast_default(ExitCode::USR_ILLEGAL_STATE, "failed to update sector infos")
```

Comment:
nit: while you're here you could use the improved `with_context_code` pattern

Reply:
I'm happy to do this in a followup, but I'm not _quite_ sure what the correct solution here is. I could use `context_code`, but `downcast_default` handles several other cases:

- Converts hamt/amt errors to the correct exit codes.
- Correctly handles USR_SERIALIZATION errors.